server/routes/index.js

```diff
@@ -4,4 +4,11 @@
 
 'use strict';
 
-module.exports = [].concat(require('./base'), require('./auth'), require('./search'), require('./visits'));
+// TODO just sniff out all the files under /routes?
```

Comment:
I was thinking something along the lines of
```js
// inside index.js
var fs = require('fs');
var routes = [];
fs.readdirSync(__dirname).forEach(function(file) {
  if (file != 'index.js') { routes.concat(require('./' + file)); }); }
});
module.exports = routes;
```
What do you think? Am I missing anything?